IshiHisashi/GlucoFit
# feat: JDEV-68 - Create log screens for Medicine, Activity, and Food

## Description of this change
- create medicine log, activity log, and food log screens
- update the bar chart on the bottom of the home screen to use actual data from DB
- update bottom nav bar, add some svgs
- styling is NOT focused yet

## Steps to test
1. `cd frontend`, `npm install`, and run the app on an emulator
2. Create any logs. 
3. After creating logs, you can check how the home screen is updated. 

<img src="https://github.com/user-attachments/assets/7be338a3-b4e6-4272-8774-cf71a4231d17" width="150px" />
<img src="https://github.com/user-attachments/assets/0a8adbf4-ee8b-4a5e-bbb0-330c0f360f7f" width="150px" />
<img src="https://github.com/user-attachments/assets/92dfed89-1548-412f-921f-3d3cf31b73d4" width="150px" />
<img src="https://github.com/user-attachments/assets/a0cd8f76-4656-4560-8c9c-48aa15b736e0" width="150px" />
<img src="https://github.com/user-attachments/assets/2ba97909-7a6e-4c84-9516-64b4d6ff11f9" width="150px" />
<img src="https://github.com/user-attachments/assets/4497830d-f326-4e92-9d0f-d9ae584dcb18" width="150px" />

## Ticket link
https://team-jupiter-capstone.atlassian.net/browse/JDEV-68
